transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/TextWidget.java

```diff
@@ -71,7 +73,27 @@ public void onFocusChange(View v, boolean hasFocus) {
                     }
                 }
             });
-            editText.addTextChangedListener(new InputMaskTextWatcher(editText));
+            editText.addTextChangedListener(new TextWatcher() {
```

Comment:
@peter-joseph if this approach solves our issue, please create a static inner class that inherits from InputMaskTextWatcher

Reply:
yep, that works great too!